draft-demarco-oauth-status-attestations.md

```diff
@@ -207,20 +243,42 @@ or values.
 
 # Proof of Possession of a Credential
 
-The concept of Proof of Possession (PoP) of a Credential within the framework of the Status Assertion specification encompasses a broader perspective than merely possessing the digital bytes of the Credential. It involves demonstrating rightful control or ownership over the Credential, which can manifest in various forms depending on the technology employed and the nature of the digital Credential itself. For instance, a Credential could be presented visually (de-visu) with a personal portrait serving as a binding element.
+The concept of Proof of Possession (PoP) of a Credential within the
+framework of the Status Assertion specification encompasses a broader
+perspective than merely possessing the digital bytes of the Credential.
+It involves demonstrating rightful control or ownership over the
+Credential, which can manifest in various forms depending on the
+technology employed and the nature of the digital Credential itself.
+For instance, a Credential could be presented visually (de-visu)
+with a personal portrait serving as a binding element.
 
-While this specification does not prescribe any additional methods for the proof of possession of the Credential, it aims to offer guidance for concrete implementations utilizing common proof of possession mechanisms. This includes, but is not limited to:
+While this specification does not prescribe any additional methods
+for the proof of possession of the Credential, it aims to offer
+guidance for concrete implementations utilizing common proof of
+possession mechanisms. This includes, but is not limited to:
 
 1. Having the digital representation of the credential (the bytes).
```

Comment:
This does not provide a proof of possession. It is only a necessary condition but not sufficient